public/app/features/dashboard-scene/serialization/layoutSerializers/RowsLayoutSerializer.ts

```diff
@@ -17,11 +17,28 @@ export function serializeRowsLayout(layoutManager: RowsLayoutManager): Dashboard
 
 export function serializeRow(row: RowItem): RowsLayoutRowKind {
   const layout = row.state.layout.serialize();
+
+  // Normalize Y coordinates to be relative within the row
```

Comment:
This seems out of place?

Reply:
@harisrozajac can you clarify why we did this here? Maybe it is a bug we should solve in a separate PR